helmfile/charts/hnc/controller/templates/user-rbac.yaml

```diff
@@ -7,6 +7,18 @@ metadata:
     {{- include "hnc.labels" $ | nindent 4 }}
   name: {{ include "hnc.fullname" $ }}-user-role
 rules:
+- apiGroups:
+  - hnc.x-k8s.io
+  resources:
+  - hierarchyconfigurations
+  verbs:
+  - get
+  - list
+  - watch
+{{- if $.Values.rbac.user.allowConfigEdit }}
+  - patch
+  - update
+{{- end }}
```

Comment:
Shouldn't this be per namespace? (Or at least the edit part?)

Reply:
Yes the edit/patch should be on a namespace level. I guess we said it is fine for them to list all namespaces and that includes hnc namepsaces

Reply:
Yes I think i will remove it